openwisp_controller/config/base/vpn.py

```diff
@@ -204,13 +245,18 @@ def save(self, *args, **kwargs):
             self._generate_wireguard_keys()
         if self.subnet and not self.ip:
             self.ip = self._auto_create_ip()
+        if self._is_backend_type('zerotier'):
+            config = deepcopy(self.config['zerotier'][0])
+            config['name'] = self.name
+            if created:
+                self._create_zerotier_server(config)
```

Comment:
Can we move this to `self.update_vpn_server_configuration`?

Reply:
Replied here: https://github.com/openwisp/openwisp-controller/pull/778#discussion_r1249786608